### PR TITLE
Groovyserv POC for develop branch

### PR DESCRIPTION
**Pull request by pibi**

### CODE REVIEW

File: test/runners/java_spec.js [hunk 1]
```diff
@@ -7,13 +7,14 @@ describe('java runner', function() {
...
-          '   static void main(String[] args){',
-          '        System.out.println("42");',
-          '   }',
-          '}'].join('\n')
+        code: [`
+          class Solution {
+             public static void main(String[] args){
+                  System.out.println("42");
+             }
+          }`].join('\n')
```
Comment (OverZealous) [inline test/runners/java_spec.js]: Minor thing, but this can drop the `[` and `].join('\n')` if we're using template strings.

File: Makefile [hunk 2]
```diff
@@ -6,7 +6,7 @@ CONTAINERS=node dotnet jvm python ruby alt rust julia systems dart crystal ocaml
 # recent containers should be updated when adding or modifying a language, so that
 # the travis build process will test it. The process cant test all languages
 # without timing out so this is required to get passed that issue.
-RECENT_CONTAINERS=node
+RECENT_CONTAINERS=java
```
Comment (kazk) [inline Makefile]: Travis build matrix is used now to test all images, so

```yml
- TEST_IMG=java
```

needs to be added in `.travis.yml`.

**Comment by pibi** [comment]
@jhoffner Ok, will update it

File: test/runners/java_spec.js (hunk 1 above)
Comment (pibi) [inline test/runners/java_spec.js]: Fixed, thanks!

**Comment by jhoffner** [comment]
@pibi looks like Travis is still failing. We will need to get that passing before I can test. I'm also leaving tomorrow on vacation for 2 weeks so no rush.

**Comment by kazk** [comment]
@jhoffner is this supposed to co-exist with `jvm-runner`? I think `jvm-runner` is failing because it's using the modified `lib/runners/java.js`.

Disable `java_spec.js` in `docker/jvm.docker`?
```diff
- RUN mocha -t 10000 test/runners/{java,clojure,groovy,scala,kotlin}_spec.js
+ RUN mocha -t 10000 test/runners/{clojure,groovy,scala,kotlin}_spec.js
```

**Comment by jhoffner** [comment]
Thanks @kazk. Yes we can remove java from the jvm docker.

**Comment by pibi** [comment]
Thanks @kazk , will remove it. To be honest I didn't pay much attention to that because there were 2 different image failing, the JVM (I tought it was using the Makefile, not Travis) and Julia. So, removing java form this one will not solve the Julia issue, this means that Travis will still fail

**Comment by pibi** [comment]
@jhoffner Should I have to disable the Julia job too?

**Pull request merged**